contracts/lz/interfaces/ILayerZeroReceiver.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.5.0;
+
+interface ILayerZeroReceiver {
```

Comment:
Please, instead of creating a new file, let's import `ILayerZeroReceiver` adding LayerZero repo as dependency